app/presenters/tree_builder_compliance_history.rb

```diff
@@ -50,18 +50,21 @@ def x_get_compliance_kids(parent, count_only)
     count_only_or_objects(count_only, kids)
   end
 
+  def get_parent_elm(parent, node)
```

Comment:
the name is misleading. The element is not parent. It's a "policy" node/element.